WeakAuras/Prototypes.lua

```diff
@@ -3994,7 +4010,19 @@ Private.event_prototypes = {
         init = "(not WeakAuras.IsRetail() and spellName and select(3, GetSpellInfo(spellName))) or (WeakAuras.IsRetail() and spellId and select(3, GetSpellInfo(spellId))) or 'Interface\\\\Icons\\\\INV_Misc_QuestionMark'",
         store = true,
         test = "true"
-      }
+      },
+      {
+        name = "spellSchoolString",
+        display = L["Spell School String"],
+        type = "string",
+        init = "spellSchool and WeakAuras.SpellSchool(spellSchool)",
+        store = true,
+        test = "true",
+        hidden = true,
+        enable = function(trigger)
+          return trigger.subeventPrefix and (trigger.subeventPrefix:find("SPELL") or trigger.subeventPrefix == "RANGE" or trigger.subeventPrefix:find("DAMAGE"))
+        end
+      },
```

Comment:
I don't know about that. Do you think it's useful enough?

Reply:
It can be useful for the combat text use case, i don't mind removing it